scripts/main.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const tbody = document.querySelector('#tbody');
+
+async function loadData(url) {
+  const response = await fetch(url);
+  const json = await response.json();
+  return json;
+}
+
+async function completeTable() {
+  const todoList = await loadData('https://jsonplaceholder.typicode.com/todos');
+  const users = await loadData('https://jsonplaceholder.typicode.com/users');
+  for (let i = 0; i < todoList.length; i++) {
+    const tr = document.createElement('TR');
+    const currentUser = users.find(user => user.id == todoList[i].userId);
```

Comment:
викоритовуй строге порівняння
```suggestion
    const currentUser = users.find(user => user.id === todoList[i].userId);
```